### PR TITLE
Feature/merge on include

### LINKED ISSUE #107: Load v1 remote recipes that have "include" in them
## Use Case
Some of the v1 recipes in the database use the keyword "include" to reference other JSON documents. We want to be able to load and pack these recipes as well. 

example: https://github.com/mesoscope/cellPACK_data/blob/master/cellPACK_database_1.1.0/recipes/BloodPlasma1.2.json

## Solution
After loading a v1 remote recipe, check for "include" keyword and also download those JSONs before converting the recipe 

## Alternatives
*Please describe any alternatives you've considered, even if you've dismissed them*

### PR DESCRIPTION
**Pull request by rugeli**
Problem
=======
What is the problem this work solves, including
we want to be able to pack recipes that include other recipes. 
e.g. `pack -r autoPACKserver/recipes/BloodPlasma1.2.json -c packing-configs/run.json`
local test recipe `pack -r cellpack/test-recipes/v1/BloodPlasma_noHIV_1.0_2D.json -c packing-configs/run.json`
closes #107 

Solution
========
What I/we did to solve this problem
- edited `pandaBullet_place` to handle remote recipes that include other recipes 
- fixed pytest for migration version 
- renamed and converted part of the variables to snake case   
- fixed path directory maker, passed tests in github action 

with @meganrm 

## Type of change
* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)

Screenshots (optional):
-----------------------
Show-n-tell images/animations here
Screenshot 2022-11-09 at 2 39 28 PM

### CODE REVIEW

**Comment by mogres** [comment]
Can you let me know of an example recipe which uses `include`? Thanks!

File: cellpack/autopack/__init__.py [hunk 1]
```diff
@@ -294,15 +272,17 @@ def checkErrorInPath(p, toreplace):
     return newpath[:-1]
 
 
-def fixOnePath(p):
-    for v in replace_path:
+def fixOnePath(path):
+    path = str(path)
+    for old_value, new_value in replace_path.items():
```
[the remark this answers is not in the document]
Comment (rugeli) [reply cellpack/autopack/__init__.py:277]: `replace_path` is defined in L224 in `autopack/__init__.py`. 
```
replace_path = {
    "autoPACKserver": autoPACKserver,
    "autopackdir": autopackdir,
    "autopackdata": appdata,
}
```

**Comment by rugeli** [comment]
Update: 

- we updated arguments for `Ingredient.pack_at_grid_pt_location`
- renamed `load_remote_file` to `lode_file`
- moved urllib import to the top
- renamed `replace_path` to `REPLACE_PATH` to make it clearer 

Thanks to @mogres 's good suggestions!

**Comment by mogres** [comment]
Run into an error while running the blood plasma packing:
image
Maybe the translation for pdbs is not added in?

**Review by meganrm — APPROVED** [review]

**Review by mogres — APPROVED** [review]

**Pull request merged**